third_party/blink/renderer/bindings/core/v8/record_replay_interface.cc

```diff
@@ -1578,6 +1579,24 @@ static void LayoutDom(
   }
 }
 
+static void ForTestingSerializeValueToString(
+  const v8::FunctionCallbackInfo<v8::Value>& args) {
+  v8::Isolate* isolate = args.GetIsolate();
+
+  CHECK(args.Length() == 1 &&
+        "must be called with a single value");
+
+  auto value = args[0];
+
+  scoped_refptr<SerializedScriptValue> serializedValue =
+    SerializedScriptValue::Serialize(
+      isolate, value,
+      SerializedScriptValue::SerializeOptions(), ASSERT_NO_EXCEPTION);
+  v8::Local<v8::String> s = ToV8String(isolate, serializedValue->ToWireString().Utf8().c_str());
```

Comment:
Not sure if this is a good idea. Technically its a string, but its really just bytes. If it works, good! If not, we might want to convert to base64 first.

Reply:
👍  yeah, I'm planning to switch to an array of the byte values instead.  will do that in a separate PR (before landing the tests)